syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -82,13 +88,13 @@ def retrieve_pointers(self):
         return pointers
 
     def __sum_shares(self, shares):
-        return sum(shares) % self.ring_size
+        return sum(shares) % self.field
```

Comment:
@abogaziah changed the name to field to be consistent with the ```AdditiveSharingTensor``` + it seems also in CrypTen they use the same name attribute

Reply:
I do not agree with that, field stands for the whole arithmetic field ]0, Z [, which makes it an incorrect and misleading name for the variable that represents Z only. also, the paper uses "ring size" so I think we should stick to it